import-io/python-connectors/import-io-simple-api/connector.py

```diff
@@ -13,24 +18,23 @@ def __init__(self, config):
         elif self.config['api_url'].startswith('https://extraction.import.io/'):
             self.api_version = 'extraction'
         else:
-            raise Exception(
```

Comment:
maybe more specifically type the Exception, not necessarily custom but just a more descriptive exception class